otherlibs/stdune-unstable/path.ml

```diff
@@ -310,6 +310,7 @@ end = struct
         match s.[i] with
```

Comment:
It might be a good idea to add a comment describing more directly what Dune considers being a "canonical" path.

Reply:
Good idea, done.